### DISCUSSION TITLE
我有个问题，这个项目这么火，为什么没上github trending，单独过滤中文语言连去年火起来后来看起来有点烂尾的北大考试试卷分享的项目都有，就是没有这个，上榜的的规则有人知道吗？

### DISCUSSION THREAD

**Discussion by daxionga** [WeNeedHome/SummaryOfLoanSuspension#865] [General]
如题

**Comment by iosyyy** [comment]
可能因为没LICENSE吧

**Reply by daxionga** [reply]
不对吧，这哥们都霸榜多少天了。  [kkndme](https://github.com/shengcaishizhan/kkndme_tianya)，也没license啊

**Reply by daxionga** [reply]
[阮大的科技周刊看起来也没有](https://github.com/ruanyf/weekly)

**Comment by kifastcall** [comment]
trending 之前就苦恼于被中文霸榜 ,
所以排序算法一直都有调整的, 
资料收集` 面试` 笔记 之类权重就是比较低的

**Reply by daxionga** [reply]
所以这个算法到底是啥样的呢
